src/pages/components/Card/Card.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Card = ({ title, description }) => (
+const Card = ({ title, description, icon }) => (
   <div
-    className={`card-container flex flex-col justify-center 
-    shadow-xl rounded-lg text-center py-3 px-5 my-10 select-none`}
+    className={`card-container  card-custom-border flex flex-col justify-center 
+    shadow-xl rounded-lg py-3 px-5 my-10 select-none`}
   >
+    <span className="rounded-full h-16 w-16 bg-white my-4 justify-center text-center">
+      <i className={icon}> </i>
```

Comment:
This can be a [self-closing tag](http://xahlee.info/js/html5_non-closing_tag.html)

Reply:
Yeah...done